apps/strapi-cms/tsconfig.json

```diff
@@ -1,7 +1,6 @@
 {
   "extends": [
     "@strapi/typescript-utils/tsconfigs/server",
-    "../../tsconfig.json"
```

Comment:
@kin0992 @datalek  i had to remove this line, and relative linting clause in `admin.ts`,  `database.ts`, `server.ts` due to this runtime error with strapi:
```
strapi-cms:dev: [2023-10-19 13:41:25.925] error: Cannot read properties of undefined (reading 'type')
strapi-cms:dev: TypeError: Cannot read properties of undefined (reading 'type')
strapi-cms:dev:     at /home/merack/workspaces/developer-portal/node_modules/@strapi/plugin-users-permissions/server/services/users-permissions.js:115:39
strapi-cms:dev:     at Array.filter (<anonymous>)
strapi-cms:dev:     at /home/merack/workspaces/developer-portal/node_modules/@strapi/plugin-users-permissions/server/services/users-permissions.js:115:10
strapi-cms:dev:     at /home/merack/workspaces/developer-portal/node_modules/lodash/lodash.js:4967:15
strapi-cms:dev:     at baseForOwn (/home/merack/workspaces/developer-portal/node_modules/lodash/lodash.js:3032:24)
strapi-cms:dev:     at /home/merack/workspaces/developer-portal/node_modules/lodash/lodash.js:4936:18
strapi-cms:dev:     at Function.forEach (/home/merack/workspaces/developer-portal/node_modules/lodash/lodash.js:9410:14)
strapi-cms:dev:     at Object.getRoutes (/home/merack/workspaces/developer-portal/node_modules/@strapi/plugin-users-permissions/server/services/users-permissions.js:108:7)
strapi-cms:dev:     at Object.getRoutes (/home/merack/workspaces/developer-portal/node_modules/@strapi/plugin-users-permissions/server/controllers/permissions.js:22:58)
strapi-cms:dev:     at dispatch (/home/merack/workspaces/developer-portal/node_modules/koa-compose/index.js:42:32)
```
This cause an annoying error that delay loadin
[Screencast from 19-10-2023 13:41:14.webm](https://github.com/pagopa/developer-portal/assets/24475319/c30d91b2-5c7f-4a6e-9c8a-4632d6d06891)
g of permission session as you can see here:


This PR is only a POC so i didn't spent effort to better resolve this issue but i think is somthing related to strict typescript configuration.